index-handlers/src/main/java/no/sikt/nva/nvi/index/model/document/PublicationChannel.java

```diff
@@ -0,0 +1,73 @@
+package no.sikt.nva.nvi.index.model.document;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.net.URI;
+import java.util.Arrays;
+
+@JsonSerialize
+public record PublicationChannel(URI id,
+                                 String type,
+                                 ScientificValue scientificValue) {
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public enum ScientificValue {
+        LEVEL_ONE("LevelOne"),
+        LEVEL_TWO("LevelTwo");
```

Comment:
Should it be possible to represent "LevelZero" as well, or is that not valid?

Reply:
That would be an illegal state for a `NviCandidate`, since `LevelOne` is a minimum requirement to qualify for nvi candidacy